### PR TITLE
Structure layout

### PR DESCRIPTION
**Pull request by georginaw**

### CODE REVIEW

File: stylesheet.css [hunk 1]
```diff
@@ -4,9 +4,190 @@
...
+	background-color: #000000;
+}
+
+.container {
+	position: relative;
+	margin: 0 auto;
+	width: 1280px;
+	background-repeat: no-repeat;
+	height: 100%;
+	background-color: #ffffff20;
```
Comment (ashleycoles) [inline stylesheet.css]: #ffffff20 isn't a valid hex code

File: stylesheet.css [hunk 2]
```diff
@@ -4,9 +4,190 @@
...
+.navbar .link:hover {
+	font-weight: bold;
+}
+
+.navbar .link.current-page {
+	font-weight: bold;
+	border-width: 2px 2px 0px 2px;
+	border-style: solid;
+	border-color: #ffffff;
+	background-color: #ffffff20;
```
Comment (ashleycoles) [inline stylesheet.css]: Same problem here, invalid hex code.

File: stylesheet.css (hunk 1 above)
Comment (georginaw) [reply stylesheet.css]: Thanks - opacity as mentioned, ok to keep it?

File: stylesheet.css (hunk 2 above)
Comment (georginaw) [reply stylesheet.css]: As above

**Pull request merged**